tools/run_tests/sanity/build_banned_constructs.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+# Copyright 2019 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+cd "$(dirname "$0")/../../.."
+
+#
+# Prevent the use of single line comments after license clauses, since this
+# chokes the internal import process.
+#
+
+git grep -EIrn '^licenses\(.*#' '{bazel,examples,src,test,tools}/**/BUILD' | diff - /dev/null
```

Comment:
1) should we include `BUILD.bazel` files?
2) This command does not work on my system. Does it work on yours? Seems like a glob expansion issue. Modify `bazel/BUILD` or `examples/cpp/route_guide/BUILD` to have a license comment, and there are no matches. Compare with `git grep -EIrn '^licenses\(.*#' 'examples/**/BUILD' | diff - /dev/null`, which does find the problems on my system.

Reply:
Requesting changes to prevent auto-merge.

Reply:
Good catch... last moment change blew up on me... fixed now, with stupider code that should hopefully be more robust. PTAL.